runtime/help/options.md

```diff
@@ -565,12 +567,14 @@ so that you can see what the formatting should look like.
     "pluginrepos": [],
     "readonly": false,
     "relativeruler": false,
+    "reload": "prompt",
     "rmtrailingws": false,
     "ruler": true,
```

Comment:
Also valid for:
https://github.com/zyedidia/micro/blob/dc186429852572fd9ce419c02822d7ca61629cb8/internal/config/settings.go#L83-L86